notifications.go

```diff
@@ -208,3 +209,55 @@ func (v *vocdoniHandler) createNotifications(electionID types.HexBytes, ownerFID
 	}
 	return nil
 }
+
+func (v *vocdoniHandler) directNotificationsHandler(msg *apirest.APIdata, ctx *httprouter.HTTPContext) error {
```

Comment:
I'd change quite a lot this method, some proposals:

1. Run the code on background, this can be a very costly operation, and it should have a throttle. The Endpoint should return 200 and the background routine would start notifications
2. Notifications should only be allowed once per day/election or something like that, so we need to store this information. We need to avoid spammy operations. 
3. Instead of going user-by-user to check it already voted, it might be better to, using mongo internal operations, make the diff between voters and census. Pretty sure chatGPT can help on that.
4. The backend should inform somehow in real-time, how many users have been already notified and how many failed (due an API error or something like that).

Reply:
5. I don't really like to give the user the option to define a list of FIDs. I'd instead allow the user to set something such as "number of users to notify" or "order by weight". But not the FIDs itself (for security reasons too).

Reply:
About make this operations in background, yes for sure, this is only a initial implementation to test the feature.
For the rest of the points, we need to talk about these product requirements, because when we design the feature, it does not work as you said.

Reply:
And the problem with the number 3 is that our database stores the usernames of the census participants (census collection) but the fids of the voters (votes collections), and the relation between both is in the user metadata (users collection).